server/constants/activities.js

```diff
@@ -31,8 +31,7 @@ export default {
   COLLECTIVE_UPDATE_CREATED: 'collective.update.created',
   COLLECTIVE_UPDATE_PUBLISHED: 'collective.update.published',
   COLLECTIVE_CONTACT: 'collective.contact',
-  COLLECTIVE_VIRTUAL_CARD_ASSIGNED: 'collective.virtualcard.assigned',
-  COLLECTIVE_VIRTUAL_CARD_CREATED: 'collective.virtualcard.created',
```

Comment:
Really small change, we have a practice of keeping past activities that are not used anymore. Because they are still used in the database for past entries.

Instead of removing them, can you move them to the bottom of the list?

Reply:
@Forster-Groux Could you do the change so we can merge?

Reply:
I did it yesterday by amending the last commit (there is no new commit). I've just repushed because I forgot the graphQL schema. It should be good now.

Reply:
Perfect.